components/ThemeSwitcher/ThemeSwitcher.tsx

```diff
@@ -46,7 +46,7 @@ export const ThemeSwitcher = () => {
 		document.addEventListener("keyup", handleThemeSwitcherKB)
 
 		return () => document.removeEventListener("keyup", handleThemeSwitcherKB)
-	}, []) // eslint-disable-line
+	}, [])
```

Comment:
I removed this suppression because I want to deal with all warnings in the future.

Reply:
How do we make sure ESLint doesn't give warnings for empty dependency arrays when we only want something to run on first render?

Reply:
That's the way to do it, but I wanted to have a clear view of all the warnings when running a lint and decide what to do.
In this case, I forgot to add the rule I was disabling.
I'll take care of them in the future.